docs/querying/api.md

```diff
@@ -936,6 +936,27 @@ $ curl http://localhost:9090/api/v1/status/tsdb
 
 *New in v2.15*
 
+
+## Remote Read API
+
+This API provides data read functionality from prometheus. This interface expects [snappy](https://github.com/google/snappy) compression.
```

Comment:
This should follow the style of the rest of the docs.

Reply:
Can you clarify what you mean by this?

Reply:
@brian-brazil Is there a particular style guide?

Reply:
There isn't (and it'd be too granular to cover this), so look at the rest of the page for how we word things.